backend/pkg/config/config.go

```diff
@@ -220,6 +223,7 @@ type Config struct {
 	OAuth         OAuth          `mapstructure:"oauth"`
 	CustomBoards  []CustomBoard  `mapstructure:"customBoards"`
 	Local         bool           `mapstructure:"local"`
+	Users         string         `mastructure:"users"`
```

Comment:
I would like to add an `Board` option around to give this users config a bit more context. Also its not used right now?

Reply:
yes you're right. I removed it.